python/examples/workflows/multi_agents_simple.py

```diff
@@ -53,4 +54,8 @@ async def main() -> None:
 
 
 if __name__ == "__main__":
-    asyncio.run(main())
+    try:
```

Comment:
If you could remove the try except block from main above, it's no longer needed with this